web-ui/src/pages/CheckinsReportPage.jsx

```diff
@@ -1,149 +1,161 @@
-      setSelectedPdls([]);
-      setFilteredPdls(pdls);
-    }
+  const handleQuarterClick = evt => {
+    /** @type {HTMLButtonElement} */
+    const button = evt.currentTarget;
+    const isNextButton = button.attributes
+      .getNamedItem('aria-label')
+      .value.includes('Next');
+    isNextButton
+      ? setReportDate(new Date(reportDate.setMonth(reportDate.getMonth() + 3)))
+      : setReportDate(new Date(reportDate.setMonth(reportDate.getMonth() - 3)));
   };
 
-  const handleClosed = () => {
-    setClosed(!closed);
-  };
+  /** @type {PDLProfile[]} */
+  const pdls = selectCheckinPDLS(state, closed, planned).sort(sortByLastName);
 
-  const handlePlanned = () => {
-    setPlanned(!planned);
-  };
+  // Set the selected PDLs to the mapped PDLs
+  useEffect(() => {
+    const mapped = selectMappedPdls(state);
+    setSelectedPdls(mapped);
+  }, [state]);
+
+  // Set the mapped PDLs to the PDLs with members
+  useEffect(() => {
+    if (!pdls) return;
+    pdls.forEach(
+      pdl => (pdl.members = selectTeamMembersWithCheckinPDL(state, pdl.id))
+    );
+    pdls.filter(pdl => pdl.members.length > 0);
+  }, [pdls, state]);
 
   return (
-    <div>
-      <div className="filter-pdls-and-members">
-        <Autocomplete
-          id="pdlSelect"
-          multiple
-          options={pdls}
-          value={selectedPdls || []}
-          onChange={onPdlChange}
-          getOptionLabel={option => option.name}
-          renderInput={params => (
-            <TextField
-              {...params}
-              label="Select PDLs..."
-              placeholder="Choose which PDLs to display"
-            />
-          )}
-        />
-        <TextField
-          label="Select employees..."
-          placeholder="Member Name"
-          value={searchText}
-          onChange={e => {
-            setSearchText(e.target.value);
-          }}
-        />
-      </div>
-      <div className="checkbox-row">
-        <label htmlFor="closed">Include closed</label>
-        <input id="closed" onClick={handleClosed} type="checkbox" />
-        <label htmlFor="planned">Include planned</label>
-        <input id="planned" onClick={handlePlanned} type="checkbox" />
+    <div className="checkins-report-page">
+      <MemberSelector
+        initialFilters={[{ type: FilterType.ROLE, value: 'PDL' }]}
+        title="Select PDLs"
+        selected={selectedPdls}
+        onChange={setSelectedPdls}
+        listHeight={180}
+        exportable
+      />
+      <Box sx={{ display: 'flex', justifyContent: 'center' }}>
+        <ButtonGroup variant="text" aria-label="Basic button group">
+          <Tooltip title="Previous quarter">
+            <>
+              <IconButton
+                disabled={true}
+                aria-label="Previous quarter`"
+                onClick={handleQuarterClick}
+                size="large"
+              >
+                <ArrowBackIcon style={{ fontSize: '1.2em' }} />
+              </IconButton>
+            </>
```

Comment:
A container element or fragment is required in order to allow Tooltip to function as it is not allowed to have a disabled child as a direct descendent.